components/text/less/manifests/sample-manifest.p5m

```diff
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright 2016 <contributor>
+# Copyright 2020 <contributor>
 #
```

Comment:
Usually we don't update the sample-manifest if only the date changes.

Reply:
Changed it back, thanks.